src/runner.ts

```diff
@@ -219,10 +220,19 @@ export class TestRunner implements vscode.Disposable {
                   }
 
                   const asText = error || "Test failed";
-                  const testMessage =
-                    actual !== undefined && expected !== undefined
-                      ? vscode.TestMessage.diff(asText, expected, actual)
-                      : new vscode.TestMessage(asText);
+                  let testMessage: vscode.TestMessage;
+
+                  if (actual !== undefined && expected !== undefined) {
+                    testMessage = vscode.TestMessage.diff(asText, expected, actual);
+                  } else if (doesTextContainAnsiCodes(asText)) {
+                    const markdownString = new vscode.MarkdownString();
+                    markdownString.supportHtml = true;
+                    markdownString.isTrusted = true;
```

Comment:
I don't think this text needs to be / should be marked as trusted since it's derived from the test runner output

Reply:
If it's not marked as trusted we can't color the output

https://github.com/microsoft/vscode/blob/6d2920473c6f13759c978dd89104c4270a83422d/src/vs/base/browser/markdownRenderer.ts#L308